app/services/apiEndpoints.js

```diff
@@ -1,4 +1,4 @@
-export const BASE_URL = API_BASE_URL;
+export const BASE_URL = API_BASE_URL; // eslint-disable-line no-undef
```

Comment:
Should we import this variable instead of referencing it globally?

Reply:
Replied on the other PR. Thanks